### ISSUE TITLE
how to speed up the response time?

### ISSUE THREAD

**Issue by zhiying8710** [Trac-Systems/tap-reader#23]
The response of the interface request provided by this client is super slow. It usually takes several minutes to get no result. For example, when requesting `http://localhost:5099/getDeployments?offset=510&max=10`, how could I possibly use this for production environment?

**Comment by TracSystems** [comment]
Hi,
sorry for the late reply. Just tested it using http://localhost:5099/docs/
and got immediate results. 

It can be you didn't get connected to enough peers, which increases the "hopping". Did you get many disconnects prior to executing?

**Comment by zhiying8710** [comment]
no, just stuck for a long time for any new(different params) request, do i need to open my port to everyone?
